### ISSUE TITLE
session not created: This version of ChromeDriver only supports Chrome version 83 (SessionNotCreated)

### ISSUE THREAD

**Issue by cremope** [rosolko/WebDriverManager.Net#122]
When trying to use a commands: 

"new DriverManager().SetUpDriver(new ChromeConfig());"
"_webDriver = new ChromeDriver();"

Displays the error "session not created: This version of ChromeDriver only supports Chrome version 83 (SessionNotCreated)"

Project:
C#
MsTest
Specflow
netcoreapp3.1

Chrome Version: 89.0.4389.72 

Can you help me?

**Comment by naeemakram** [comment]
The following code works fine for me, it is downloading the latest version of Chrome Driver automatically. 
Which version of Chrome is installed on your machine right now, is it updated? 

```
DriverManager webDriverManager = new DriverManager();
webDriverManager.SetUpDriver(new ChromeConfig());

var chromeDriver = new ChromeDriver();

chromeDriver.Navigate().GoToUrl("http://www.google.com");
Assert.True(chromeDriver.Title.Contains("Google"));

chromeDriver.Quit();
```

**Comment by josefsirucka** [comment]
I have the same Issue now:

session not created: This version of ChromeDriver only supports Chrome version 88 Current browser version is 90.0.4430.93 with binary path C:\Program Files (x86)\Google\Chrome\Application\chrome.exe (SessionNotCreated)

Chrome driver installed: 90.0.4430.93
WebDriverManager 2.11.1

**Comment by kirankumarsalamkayala** [comment]
Hi,
Receiving same error as stated above on Chrome browser version 90.0
any solution please

**Comment by kirankumarsalamkayala** [comment]
@josefsirucka 
I could resolve this issue by un installing Driver packages that i was using  and deleting the existing driver.exe files , try at your end this should work.

**Comment by josefsirucka** [comment]
I have no driver package installed - and I have tried to delete existing exe driver, and the situation is the same.

**Comment by aasimahmed** [comment]
Pass in the Directory of the installed ChromeDriver returned through webDriverManager.SetupDriver(); to the first argument of new ChromeDriver();

https://github.com/SeleniumHQ/selenium/blob/trunk/dotnet/src/webdriver/Chrome/ChromeDriver.cs#L93 - Which takes the Directory as the first argument.

**Comment by cremope** [comment]
Sorry for the delay guys!

I was only able to get back to the subject now, and I managed to resolve it.
What I did was include the DriverManager call inside the new ChromeDriver().

In this way:

>     private static IWebDriver _webDriver;
>     _webDriver = new ChromeDriver(new DriverManager().SetUpDriver(new ChromeConfig()));

    
Remembering, it was necessary to update the Google Chrome browser for this.

It may be that your browser is about to update, and the SetUpDriver() method defaults to always getting the latest version of chromeDriver. 

Unless you change the version by passing it as a parameter.

I hope to help others with this tip.

Hugs
